text/0007-weekly-inspo.md

```diff
@@ -0,0 +1,301 @@
+---
+rfc pr: https://github.com/UCLA-Creative-Labs/cl-rfcs/pull/11
+tracking issue: https://github.com/UCLA-Creative-Labs/cl-rfcs/issues/7
+---
+
+# xplr
+
+`xplr` is a repository dedicated to tracking the concepts, blog posts, and ideas
+that inspire the Creative Labs dev team.
+
+## Blurb
+
+`xplr` will act as a central hub for all of our inspiration. Every week, members of 
+the dev team will make a pull request to the `xplr` repository containing something 
+that inspired them. Their inspiration can be anything from a cool CSS property, an 
+interesting design, quirky libraries, or exciting frameworks. During our dev meeting, 
+we will talk about the stuff we found and at the end of the discussion, update a 
+`CHANGE_LOG` to keep a weekly timeline of the inspo we added.
+
+The pull request will be in the form of a markdown file that contains a description of 
+the inspo, why they think it's interesting, and how we could potentially use it in the 
+future. The file must also include metadata to help tag and catalog who the inspo came 
+from and what category the inspo belongs to.
+
+All these markdown files will accumulate into a blog site that is easily accessible to
+anyone outside of the dev team to read. The site will be searchable, filterable, and 
+creatively designed to show off our inspo in an unique fashion.
```

Comment:
```suggestion
creatively designed to show off our inspo in a unique fashion.
```

nit: the nittiest of nits but it's "a unique" (at least in american english)

Reply:
Fanks!